docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -31,7 +31,13 @@ CREATE [ EXTERNAL ] TABLE [ IF NOT EXISTS ] table_identifier
     [ COMMENT table_comment ]
     [ PARTITIONED BY ( col_name2[:] col_type2 [ COMMENT col_comment2 ], ... ) 
         | ( col_name1, col_name2, ... ) ]
-    [ ROW FORMAT row_format ]
+    [ ROW FORMAT SERGE serde_name [WITH SERDEPROPERTIES (k1=v1, k2=v2, ...)] ]
```

Comment:
SERDE

Reply:
Thanks @srowen .
yes,we have them defined in Spark. A simple example as:
```
CREATE TABLE test2(
name STRING
)
ROW FORMAT SERDE 
'org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe' 
```